docs/source/articles/orders-and-items.html.md

```diff
@@ -20,6 +20,55 @@ The `Order` model and its embedded items identify the consumer and the items tha
 - Zero or more promo codes, which may affect item and order pricing
 - The calculated order-level price totals
 
+## Order Identification
+
+When an order is created or instantiated, it is assigned a randomized 5-digit uppercase hexadecimal value as its ID. This is so order numbers are easy to read and identify, and will play nicely on a distributed database system such as MongoDB. This ID is used throughout the system to link orders to their Payment and Shipping information, as well as in the admin for display purposes. Some clients may want this display ID to be sequentially numbered, which is possible to customize, but we strongly advise against it. MongoDB is a distributed database, so in production there are many instances of Mongo running to make sure that your app's data is not only within reach, but can be retrieved efficiently. Because of this, **it is impossible to guarantee that two simultaneous writes for the same data will result in two synchronous operations within the database itself**.
```

Comment:
Only the primary MongoDB replica set member receives writes. 

Tom, I don't think that you were in our IT discussions about this, but the only issue with the plan below is that there may be gaps in the sequence of order numbers, but not duplicates. All merchants that have had this implemented have been fine with some sporadic gaps.

Reply:
It's also important to note that the sequence may or may not follow the chronology of when the orders were placed - order 10001 may be placed after order 10002.

Reply:
Also, "easy to read and identify" is exactly why they want sequential numbers, not the random string ones.

Reply:
I guess I was trying to emphasize that our order numbers are the way they are _because_ they are easier to read and identify than a traditional BSON ID. Otherwise, what's the point? Why not just use a BSON ID?

I'll make the changes around sporadic gaps rather than simultaneous writes, and notate that the orders might not be placed in sequential order.